app/api-handlers/multiple-error-helper.js

```diff
@@ -119,7 +119,7 @@ var errorDataHelper = function(errorData) {
                 }
             }
 
-            let values = errorData.map(i => `${i.fieldName}: ${i.errorValue != null ? i.errorValue : ''}`).join(", ");
+            let values = errorData.map(i => `${i.fieldName}: ${i.errorValue !== null ? i.errorValue : ''}`).join(", ");
```

Comment:
Could you double check that this doesn't adversely effect the display of missing data?  Got a sneaking suspicion that the errorValue may have used the String 'null'.  Once gotcha of the linter rules I suppose - the original intention was to cover both null && 'null' with minimal fuss.